pkg/plugins/trivy/plugin.go

```diff
@@ -1543,6 +1545,24 @@ func GetScoreFromCVSS(CVSSs map[string]*CVSS) *float64 {
 	return nvdScore
 }
 
+func GetVectorFromCVSS(CVSSs map[string]*CVSS) string {
```

Comment:
This could be tricky as there could be several `cvss vendors` and it could lead to different `Severity` decisions than trivy.
trivy sets the priority to the vendor by package, for example, if the package is `rpm` then the priority goes to `redhat` example for trivy cvss output:

```
"CVSS": {
            "ghsa": {
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "nvd": {
              "V2Vector": "AV:N/AC:L/Au:N/C:C/I:C/A:C",
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V2Score": 10,
              "V3Score": 9.8
            },
            "redhat": {
              "V2Vector": "AV:N/AC:M/Au:N/C:P/I:N/A:N",
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
              "V2Score": 4.3,
              "V3Score": 5.3
            }
          },```

Reply:
Hi! Thanks for your reply! That's why I'v made it totally like in GetScoreFromCVSS and we can get Score/Vector (both) from NVD (if there is no Vendor) or from Vendor (it the data is present). Or you mean, that Vector from Vendor is not always present as field and it's possible to get Score from Vendor and Vector from NVD?

Reply:
Yes , you could get many scores from `nvd` and few `vendors` on the same scan result as in example above.
 today `trivy` know how to take the right decision and determine the `severity` (by score) taking under consideration the vulnerable pkg , choosing one vendor over the other could have different calculation which will impact the severity:
```
0.0 = None
0.1-3.9 = Low
4.0-6.9 = Medium
7.0-8.9 = High
9.0 - 10.0 = Critical


```

Reply:
Got it. One more question, in my PR I did't make any changes to `GetScoreFromCVSS(CVSSs map[string]*CVSS)` which (if I'm right) will always take the last Vendor score if it's present:
```
for name, cvss := range CVSSs {
		if name == "nvd" {
			nvdScore = cvss.V3Score
		} else {
			vendorScore = cvss.V3Score
		}
	}
```
I just copied that logic for Vector (because some people need this info in their Vuln Man process). I can change it if you point me on the result you want to get. Thanks! )

Reply:
Trivy uses some logic to determine the correct `cvss.V3score` to choose from , he choose the vendor by pkg type,
meaning if it is a `rpm pkg` then the `redhat` advisory would take advantage